cpu/stm32f4/periph/cpuid.c

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright (C) 2014 FU Berlin
```

Comment:
Also, for changes like this do you prefer to keep it in a single commit or make a new commit?